tools/c7n_gcp/c7n_gcp/resources/kms.py

```diff
@@ -100,6 +102,16 @@ def get(client, resource_info):
             return client.execute_command('get', {'name': name})
 
 
+@KmsCryptoKey.filter_registry.register('iam-policy')
+class KmsCryptokeyIamPolicyFilter(IamPolicyFilter):
+    """
+    Overrides the base implementation to process KMS Cryptokey resources correctly.
+    """
+    def _verb_arguments(self, resource):
+        verb_arguments = SetIamPolicy._verb_arguments(self, resource)
+        return verb_arguments
+
+
```

Comment:
These look like changes from the other PR. Let's try to remove any of those from this PR if we can to keep the scope as small as possible.